src/main/java/appeng/client/render/cablebus/CableBusModelLoader.java

```diff
@@ -18,7 +18,6 @@ public CableBusModelLoader(PartModels partModels) {
 
     @Override
     public void onResourceManagerReload(IResourceManager resourceManager) {
-        CableBusBakedModel.clearCache();
```

Comment:
Does should still be  required. Reloading the resources could cause the model geometry to change. Without clearing the cache, it can still return old models.

Reply:
No because the model object itself is re-created on resource reload, and the cache is no longer static.